data/categories.txt

```diff
@@ -2632,41 +2632,45 @@ sw:Bandali|kivuko
 fa:ترمینال کِشتی
 
 amenity-taxi|@transport
-en:3Taxi|Taxi Stand|Taxi Rank|U+1F695|U+1F696
-ru:3Такси
-bg:3такси|шофьор
-ar:سيارة أجرة|تاكسي|مواصلات|مواصلات
-cs:3Taxi
-da:3Taxi|taxa
-nl:3Taxi
-fi:3Taksi
-fr:3Taxi
-de:3Taxi|Taxistand|Taxe|ÖPNV
+en:3Taxi Stand|Taxi Rank|Taxi|U+1F695|U+1F696
+ar:سيارة أجرة|تاكسي|مواصلات|مواصلات|محطة التاكسي
+be:Стаянка таксі
```

Comment:
The single word "taxi" should stay as a synonym everywhere, in addition to localized single word "taxi" and localized "taxi stand" translation.

Reply:
Is it really necessary to explicitly add "taxi" to all languages? If `en:3Taxi` is present searching for "taxi" should work in all languages or?

Reply:
@vng can you please confirm that en synonyms are always used by the search?

Reply:
Confirm